share/spice/what3words/what3words.js

```diff
@@ -56,10 +66,60 @@
                         }
                     }
                 });
+            }).fail(function() {
+                    consose.log("error");
             });
+            
         });
-    };
-
+        }
+        else {
+            DDG.require('maps', function () {
+                // the object returned when the api cannot match the three word phrase
+                $.getJSON('/js/spice/what3words_standard_blend/' + address, function(data) {
+                    var item;
+                    // built-in flags use 'uk' instead of 'gb' that standardblend uses
+                    for (item in data.blends) {
+                        if (data.blends[item].country == 'gb'){
+                            data.blends[item].country = 'uk'
+                        }
+                    }
+                    Spice.add({
+                        id: 'what3words',
+                        name: 'Standardblend',
+                        model: 'Place',
+                        view: 'Places',
+                        meta: {
+                            itemType: 'suggestions',
+                            searchTerm: address
+                        },
+                        templates: {
+                            group: 'text',
+                            variants: {
+                                tile: 'video'
+                            }
+                        },
+                        data: data.blends,
+                        normalize: function(item){
+                            return {
+                                name: item.words,
+                                lat: item.geometry.lat,
+                                lon: item.geometry.lng,
+                                title: item.words,
+                                description: 'near ' + item.place + ', ' + country_map[item.country],
+                                displayLatLon: item.geometry.lat + '\xB0, ' + item.geometry.lng + '\xB0',
+                                place: item.place,
+                                icon: DDG.settings.region.getLargeIconURL(item.country)
+                                }
+                            }
+                        });
+                   
+                    
+                }).fail(function() {
+                    console.log("error");
```

Comment:
Actually, I don't think this request can fail. The data might come back as undefined though. Please confirm.